src/data/faq.json

```diff
@@ -297,11 +297,9 @@
                         "anchor": "ENError13",
                         "active": true,
                         "textblock": [
-                            "The Error 13 is thrown by Apple's Exposure Notification Framework, which is used by the Corona-Warn-App.",
-                            "It will appear if more than 6 exposure checks happened in the last 24h. Apple does not allow more than 6 exposure checks within 24h.",
-                            "The error will disappear after 24h at the latest. You don't have to do anything.",
-                            "Please do not remove and reinstall the Corona-Warn-App, because the error is caused by the underlying Apple ENF, it will persist, even after reinstallation.",
-                            "If the Error doesn't disappear after 24h, please open a issue on <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/' target='_blank' rel='noopener noreferrer'>GitHub</a>."
+                            "The Error 13 is thrown by Apple's Exposure Notification Framework, which is used by the Corona-Warn-App. It will appear if more than 6 exposure checks happened in the last 24 hours. Apple does not allow more than 6 exposure checks within 24 hours. The error will disappear after 24 hours at the latest. You don't have to do anything.",
+                            "Please do not remove and reinstall the Corona-Warn-App, because the error is caused by the underlying Apple Exposure Notification Framework which is independent from the app. Therefore, it will persist, even after reinstallation.",
+                            "If the Error doesn't disappear after 24h, please open an issue on <a href='https://github.com/corona-warn-app/cwa-app-ios/issues/' target='_blank' rel='noopener noreferrer'>GitHub</a>."

```

Comment:
We currently have some users reporting that the Error 13 doesn't disappear after 24h (see https://github.com/corona-warn-app/cwa-app-ios/issues/2395).

You should internally decide wether you want a new issue for every case or if we focus on one of the issues in the above linked overview. 
If we focus on one, we should link to this one.

Reply:
there's still not enough information about this so that we can safely link to one issue I think. A new issue for every case is still preferred.

If we get new information this week, I'll update this issue and merge with high priority.

Reply:
Okay thank you @heinezen!